domain_test.go

```diff
@@ -0,0 +1,71 @@
+package userconfig_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/giantswarm/generic-types-go"
+	"github.com/giantswarm/user-config"
+)
+
+func TestUnmarshalValidDomains(t *testing.T) {
```

Comment:
All of these tests should not be in here. Especially not import our private repos ;)